osc/commands/fork.py

```diff
@@ -56,6 +56,11 @@ def init_arguments(self):
             help="Name of the org owning the fork",
         )
 
+        self.add_argument(
+            "--git-branch",
+            help="Name of the new git branch",
+        )
+
```

Comment:
I'd move it below `self.add_argument_new_repo_name()` to achieve the following  order: org, repo, branch